BuildAddOn.py

```diff
@@ -145,7 +165,7 @@ def DownloadAndUnzip (url, dest):
                 zip.extractall (path=dest)
     elif platform.system () == 'Darwin':
         if tarfile.is_tarfile (filePath):

```

Comment:
I guess this tarfile extractor part was not run with the public devkits since they are not uploaded as tar.gz after all... But let's just leave it like that for the future.